common_issues.md

```diff
@@ -29,11 +43,22 @@ Check your INT wire, there is either a bad connection or you have it connected t
 
 ## The trackers are connected to my wifi but don't turn up on SlimeVR
 
-This is most likely a firewall issue, you'll need to go to your firewall settings and grant permission to the SlimeVR server to make sure it can access them.
+Check that you do not have two copies of the SlimeVR server running, as only one of them will show trackers connected.
+
+If only one server is running, this is most likely a firewall issue, go to SlimeVR Server folder and run `firewall.bat` as administrator.
+
+If you are still having trouble, try manually adding the SlimeVR Server to your firewall. 
+
+1. Go to **Settings** > **Network & Internet** then click on the text link **Window Firewall** (you may have to scroll down). 
+1. In the firewall window, click the link **Allow an app through firewall**.
+1. Click the **Allow another app...** button, and then **Browse...** in the opened **Add an app** window. If your options are greyed out in the **Allowed apps** window, click the **Change Settings** button to allow changes.
+1. In the **File name** text box, type `*.*` and press enter before navigating to `slimevr.jar` in your SlimeVR server folder and select it (if you cannot see files in this folder, try typing `*.*` and pressing enter again to show all files).
+1. Click the **Add** button to add the file to your firewall settings.
+1. Finally, make sure both public and private check boxes are selected in the **Allowed apps** window before clicking **OK** to save the changes.
 
 ## The trackers are connected to the SlimeVR server but aren't turning up on Steam
```

Comment:
This can not be a firewall issue, if trackers are in SlimeVR, but not in SteamVR, it's either driver issue, or order of launch issue.

Reply:
Edited to show these changes